docs/sources/installation/helm/monitor-and-alert/with-grafana-cloud.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Configure Monitoring and Alerting of Loki Using Grafana Cloud
+menuTitle: Monitor Loki with Grafana Cloud
+description: setup monitoring and alerts for Loki using Grafana Cloud
+aliases:
+  - /docs/installation/helm/monitoring/with-grafana-cloud
+weight: 100
+keywords:
+  - monitoring
+  - alert
+  - alerting
+  - grafana cloud
+---
+
+# Configure Monitoring and Alerting of Loki Using Grafana Cloud
+
+This topic will walk you through using Grafana Cloud to monitor a Loki installation that is installed with the Helm chart. This approach leverages many of the chart's _self monitoring_ features, but instead of sending logs back to Loki itself, it sends them to a Grafana Cloud Logs instance. This approach also does not require the installation of the Prometheus Operator and instead sends metrics to a Grafana Cloud Metrics instance. Using Grafana Cloud to monitor Loki has the added benefit of being able to troubleshoot problems with Loki when the Helm installed Loki is down, as the logs will still be available in the Grafana Cloud Logs instance.
+
+**Before you begin:**
+
+- Helm 3 or above. See [Installing Helm](https://helm.sh/docs/intro/install/).
+- A Grafana Cloud account and stack (including Cloud Grafana, Cloud Metrics, and Cloud Logs)
+- [Grafana Kubernetes Monitoring using Agent](/docs/grafana-cloud/kubernetes-monitoring/configuration/config-k8s-agent-guide/) configured for the Kubernetes cluster
+- A running Loki deployment installed in that Kubernetes cluster via the Helm chart
+
+**Prequisites for Monitoring Loki:**
+
+It is necessary to setup the Grafana Kubernetes Integration following the instructions in [Grafana Kubernetes Monitoring using Agent](/docs/grafana-cloud/kubernetes-monitoring/configuration/config-k8s-agent-guide/) as this will install necessary components for collecting metrics about your Kubernetes cluster and sending them to Grafana Cloud. Many of the dashboards installed as a part of the Loki integration rely on these metrics.
```

Comment:
```suggestion
You must setup the Grafana Kubernetes Integration following the instructions in [Grafana Kubernetes Monitoring using Agent](/docs/grafana-cloud/kubernetes-monitoring/configuration/config-k8s-agent-guide/) as this will install necessary components for collecting metrics about your Kubernetes cluster and sending them to Grafana Cloud. Many of the dashboards installed as a part of the Loki integration rely on these metrics.
```